adapter/PreciceInterface.c

```diff
@@ -587,9 +601,8 @@ void PreciceInterface_FreeData( PreciceInterface * preciceInterface )
 		free( preciceInterface->xloadIndices );
 	}
     
-    	if ( preciceInterface->xforcIndices != NULL ) {
-        	free( preciceInterface->xforcIndices );
-    	}
+    if ( preciceInterface->xforcIndices != NULL )
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).